projects/kubernetes-sigs/image-builder/Makefile

```diff
@@ -95,8 +95,11 @@ else ifeq ($(IMAGE_FORMAT),raw)
 		# When building Ubuntu OVA for release, we also want to fetch Bottlerocket OVA in the
 		# same job because they are usually tied together with an EKS-A release so we want them
 		# to be uploaded to the same location to be picked up by the release process
-		S3_TARGET_PREREQUISITES=$(FINAL_RAW_IMAGE_PATH) $(FINAL_BOTTLEROCKET_RAW_PATH)
-		RELEASE_TARGETS=release-raw-ubuntu-2004-efi download-bottlerocket-raw upload-artifacts-raw upload-bottlerocket-raw
+		S3_TARGET_PREREQUISITES=$(FINAL_RAW_IMAGE_PATH)
+		RELEASE_TARGETS=release-raw-ubuntu-2004-efi upload-artifacts-raw
+		ifneq ($(RELEASE_BRANCH),1-20)
+			BUILD_RAW_TARGETS+=download-bottlerocket-raw $(FINAL_BOTTLEROCKET_RAW_PATH) upload-bottlerocket-raw
```

Comment:
We already have this on line 78, maybe this should be `RELEASE_TARGETS` instead. Also, I think we need a space after `+=`

Thanks for catching this.

Reply:
Ohh good catch! lol copy-paste mistake, my bad

Reply:
Also I feel that space is not mandatory, for example, https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/kubernetes-sigs/image-builder/Makefile#L31
I guess Make is forgiving in this aspect.

Reply:
ohh Make is weird